ml-dsa/src/lib.rs

```diff
@@ -192,6 +192,11 @@ impl<P: MlDsaParams> KeyPair<P> {
     pub fn signing_key(&self) -> &SigningKey<P> {
         &self.signing_key
     }
+
+    /// The verifying key of the key pair
+    pub fn verifying_key(&self) -> &VerifyingKey<P> {
```

Comment:
I noticed there wasn't an inherent method for this and it was being accessed exclusively through `signature::Keypair` which I found a little surprising.